blockstore/apps/bundles/tests/test_storage.py

```diff
@@ -15,8 +15,8 @@ class _MockS3backend:
     A fake replacment for S3Boto3Backend in these tests.
     """
     def __init__(self, **kwargs):
-        self.access_key = kwargs.get("access_key", getattr(settings, 'AWS_S3_ACCESS_KEY_ID', None))
-        self.secret_key = kwargs.get("secret_key", getattr(settings, 'AWS_S3_SECRET_ACCESS_KEY', None))
+        self.access_key = kwargs.get("access_key", getattr(settings, 'AWS_ACCESS_KEY_ID', None))
+        self.secret_key = kwargs.get("secret_key", getattr(settings, 'AWS_SECRET_ACCESS_KEY', None))
```

Comment:
FYI @bszabo -- this typo didn't really matter since it was just in tests, but I've fixed it here so [it won't confuse us anymore](https://openedx.slack.com/archives/C02QN50TYMD/p1655323621688999?thread_ts=1655234358.696019&cid=C02QN50TYMD).

Reply:
Thanks